src/main.ts

```diff
@@ -16,7 +16,27 @@ import { debounceTime } from "rxjs/operators";
 import { HTTPService } from "@project-sunbird/ext-framework-server/services";
 import * as os from "os";
 const startTime = Date.now();
-let envs = {};
+let envs: any = {};
+const { session } = require('electron')
+const setSecurityPolicy = () => {
+  const whitelistUrls = envs.whiteListUrls.join(' ');
+  session.defaultSession.webRequest.onHeadersReceived((details, callback) => {
+    callback({
+      responseHeaders: {
+        ...details.responseHeaders,
+        'Content-Security-Policy': [`
+        script-src 'self' 'unsafe-eval' 'unsafe-inline' ${envs.APP_BASE_URL} ${whitelistUrls};
+        script-src-elem 'self' 'unsafe-eval' 'unsafe-inline' ${envs.APP_BASE_URL} ${whitelistUrls};
+        img-src * blob: data:;
+        frame-src 'self' ${envs.APP_BASE_URL} blob: ${whitelistUrls};
```

Comment:
frame-src restricts which iframe can get loaded. So please check to see if this is mandatory to set. coz this could potentially mean we cannot include any iframes other than whitelisted ones

Reply:
if we dont set any fields then it will default to script-src or default-src.